frameworks/Swift/vapor/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# install swiftenv for managing Swift versions
+eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/02090c7ede5a637b76e6df1710e83cd0bbe7dcdf/swiftenv-install.sh)"
```

Comment:
I pulled down  your repo and tested it, and that line doesn't install swift (or it's not installing correctly, at least). When it gets down to line 10 ("swift build -c release"), it says "swift: command not found"